<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PetProfile</class>
 <widget class="QDialog" name="PetProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>805</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(204, 124, 169);
color: rgb(239, 207, 187);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(239, 207, 187);
border: 2px solid  white; 
     border-radius: 3px;</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="animalDisplay">
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>700</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>700</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="backToSearchButton">
          <property name="font">
           <font>
            <family>Dyuthi</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Go Back to Search!</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <family>Dyuthi</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(239, 207, 187);
border: 2px solid  white; 
     border-radius: 3px;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_name">
        <property name="font">
         <font>
          <family>Dyuthi</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_breed">
        <property name="font">
         <font>
          <family>Dyuthi</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Breed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_age">
        <property name="font">
         <font>
          <family>Dyuthi</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Age</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_color">
        <property name="font">
         <font>
          <family>Dyuthi</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_hypo">
        <property name="font">
         <font>
          <family>Dyuthi</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Hypoallergenic</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_location">
        <property name="font">
         <font>
          <family>Dyuthi</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Location</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_shelter">
        <property name="font">
         <font>
          <family>Dyuthi</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Shelter Owner</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_phone">
        <property name="font">
         <font>
          <family>Dyuthi</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Phone</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_email">
        <property name="font">
         <font>
          <family>Dyuthi</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="descriptionBox">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>300</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
